scripting/get5.sp

```diff
@@ -446,6 +451,22 @@ public void OnPluginStart() {
       "get5_phase_announcement_count", "5",
       "The number of times Get5 will print 'Knife' or 'Match is LIVE' when the game starts. Set to 0 to disable.");
 
+  g_DemoUploadUrlCvar = CreateConVar(
+      "get5_demo_upload_url", "",
+      "If defined, it is the URL at which a server resides to accept connections to upload demos. Requires SteamWorks extension.");
+
+  g_DemoUploadAuthKeyCvar = CreateConVar(
+      "get5_demo_upload_header_key", "Authorization",
+      "If defined, it is the authorization key that is appended to the HTTP request. Requires SteamWorks extension.");
+
+  g_DemoUploadAuthValueCvar = CreateConVar(
+      "get5_demo_upload_header_value", "",
+      "If defined, it is the authorization value that is appended to the HTTP request. Requires SteamWorks extension.");
+
+  g_DemoUploadDeleteAfterCvar =
+      CreateConVar("get5_demo_upload_delete_after_upload", "0",
```

Comment:
Maybe something like `get5_demo_upload_autodelete` is bit better so we don't have `upload` twice